stdlib/tempfile.pyi

```diff
@@ -318,6 +324,13 @@ class SpooledTemporaryFile(IO[AnyStr]):
     def fileno(self) -> int: ...
     def flush(self) -> None: ...
     def isatty(self) -> bool: ...
+    if sys.version_info >= (3, 11):
+        def readable(self) -> bool: ...
```

Comment:
```suggestion
```

Reply:
Thanks, applied this and updated the comments.